jwst/resample/resample_spec_step.py

```diff
@@ -265,8 +265,10 @@ def _process_slit(self, input_models):
         else:
             result.meta.resample.pixel_scale_ratio = resamp.pixel_scale_ratio
         result.meta.resample.pixfrac = self.pixfrac
+
         self.update_slit_metadata(result)
-        update_s_region_spectral(result)
+        if result.meta.exposure.type.lower() != 'mir_lrs-fixedslit':
+            update_s_region_spectral(result)
 
```

Comment:
Per my other comment, I assume the reason you put this `if` statement here is because the LRS data in question also follow this path.  Could put the call to `find_miri_lrs_sregion(input_models[0], result.meta.wcs)` here instead?  It appears to me that all the necessary inputs are already here, and it would make it so all spectral data get their `s_region` updated in the same place